Bio/SeqUtils/ProtParamData.py

```diff
@@ -8,13 +8,245 @@
 # Turn black code style off
 # fmt: off
 
-# Kyte & Doolittle index of hydrophobicity
+# Hidrophobicity
+
+# 1 Kyte & Doolittle index of hydrophobicity
```

Comment:
Are the numbers (1 to 27) meaningful? If you can avoid this is will make any future additions easier.

Also perhaps do these in key order?

Reply:
I was doing it based on the Hidrophilicity and Surface accessibility values below, but the numbers are not meaningful by themselves. Will remove them.

By key order you mean the string in the third comment line? I believe they already are in this order with the exception of the `KyteDoolitle` scale, which is first because it is the default option, but i can also change it.

Reply:
The current order makes sense then 👍